app/views/form/page.html.erb

```diff
@@ -1,8 +1,5 @@
-<% previous_page = form.previous_page(page_key) %>
 <% content_for :before_content do %>
-  <%= govuk_back_link href: "/case_logs/#{@case_log.id}/#{previous_page}" do %>
-    Back
-  <% end %>
+  <%= link_to 'Back', 'javascript:history.back()', class: "govuk-back-link" %>
```

Comment:
@roobottom if this button is just going to replicate the browser back button does it actually make any sense to have it at all?